distribution/src/main/release/samples/pom.xml

```diff
@@ -261,24 +261,24 @@
                     <version>1.3.1</version>
                 </dependency>
                 <dependency>
-                    <groupId>javax.xml.ws</groupId>
-                    <artifactId>jaxws-api</artifactId>
-                    <version>2.3.0</version>
+                    <groupId>jakarta.xml.ws</groupId>
```

Comment:
This guy brings `jakarta.annotation-api`, should we get rid of `javax.annotation-api` in this case?